From 8e213c8996f720d3d8475b291411b22c988a74c2 Mon Sep 17 00:00:00 2001
From: Adam YH Lee <adam.yh.lee@gmail.com>
Date: Mon, 19 Jan 2015 16:53:53 -0800
Subject: [PATCH 25/30] omap3: overo: Enable McBSP2 for all Overo COMs

Both Gumstix Overo and Overo Storm COMs use TWL4030 audio module
connected to the McBSP2.  As such, enable the McBSP2 module in the
common device tree file, omap3-overo-base.dtsi, rather than in the
processor-specific device tree files, omap3-overo.dtsi and
omap3-overo-storm.dtsi.  This corrects audio on the Storm COMs where
the setting was accidentally missing from the device tree.

Signed-off-by: Ash Charles <ashcharles@gmail.com>
---
 arch/arm/boot/dts/omap3-overo-base.dtsi | 4 ++++
 arch/arm/boot/dts/omap3-overo.dtsi      | 4 ----
 2 files changed, 4 insertions(+), 4 deletions(-)

diff --git a/arch/arm/boot/dts/omap3-overo-base.dtsi b/arch/arm/boot/dts/omap3-overo-base.dtsi
index d36bf0250a05..9842650c5779 100644
--- a/arch/arm/boot/dts/omap3-overo-base.dtsi
+++ b/arch/arm/boot/dts/omap3-overo-base.dtsi
@@ -219,3 +219,7 @@
 	pinctrl-0 = <&uart2_pins>;
 };
 
+&mcbsp2 {
+    status = "okay";
+};
+
diff --git a/arch/arm/boot/dts/omap3-overo.dtsi b/arch/arm/boot/dts/omap3-overo.dtsi
index 69ca7c45bca2..932a02ff552a 100644
--- a/arch/arm/boot/dts/omap3-overo.dtsi
+++ b/arch/arm/boot/dts/omap3-overo.dtsi
@@ -32,7 +32,3 @@
 		>;
 	};
 };
-
-&mcbsp2 {
-	status = "okay";
-};
-- 
2.1.4

